e2e/images_test.go

```diff
@@ -73,6 +79,22 @@ func TestImageListQuiet(t *testing.T) {
 	})
 }
 
+func TestImageListDigests(t *testing.T) {
+	runWithDindSwarmAndRegistry(t, func(info dindSwarmAndRegistryInfo) {
+		cmd := info.configuredCmd
+		dir := fs.NewDir(t, "")
```

Comment:
I wonder if this `dir` is really used in the test? 🤔

Reply:
It's a copy paste of the other tests. Possible that it's not needed in any of them tbh. A local run of the tests implies that it's not